### PR TITLE
beakerlibtests: add new test

### PR DESCRIPTION
**Pull request by RHEmployee**
Add another test which seems suitable to upstream.

### CODE REVIEW

File: tests/beakerlib/variables-support-in-profiles/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/tuned/Sanity/variables-support-in-profiles
+#   Description: variables support in profiles
+#   Author: Branislav Blaskovic <bblaskov@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright Red Hat
```
Comment (yarda) [inline tests/beakerlib/variables-support-in-profiles/runtest.sh:11]: The same apply as in #384.

**Review by yarda — CHANGES_REQUESTED** [review]
Please fix the test not to fail on epel-8 and centos-stream.

**Comment by RHEmployee** [comment]
Hello. Thank you for your support. Should be sovlved now.

**Pull request merged**